src/access/AccessControl.cpp

```diff
@@ -61,6 +61,77 @@ bool CheckRequestPrivilegeAgainstEntryPrivilege(Privilege requestPrivilege, Priv
     return false;
 }
 
+char GetAuthModeStringForLogging(AuthMode authMode)
+{
+    switch (authMode)
+    {
+    case AuthMode::kNone:
+        return 'n';
+    case AuthMode::kPase:
+        return 'p';
+    case AuthMode::kCase:
+        return 'c';
+    case AuthMode::kGroup:
+        return 'g';
+    }
+    return 'u';
+}
+
+char * GetCatStringForLogging(char * buf, size_t size, const CATValues & cats)
+{
+    VerifyOrDie(size >= 16);
```

Comment:
Seems like a good place to pass a FixedSpan if you want to know that the buffer will be a certain size.  And certainly so given there's only one call.

This is just a style nit though.

Reply:
I'm seeing just one use of FixedSpan in the code, does that seem right?

Anyways, it doesn't have to be a fixed size, it just has to be a reasonable minimum, and will use more if provided.

Reply:
This is just a nit.   No need to add any changes.

My observation is essentially that there is a runtime assert here, when in fact we could have a compile-time check.

Reply:
I shuffled it slightly to remove the verify. (This was just a nicety to ensure we could put at least a couple characters, the slight tweaks make it handle even those cases.)

Reply:
Thanks for the amendment.  This addresses my concern.